x/or/settlement/module_simulation.go

```diff
@@ -0,0 +1,61 @@
+package settlement
+
+import (
+	"math/rand"
+
+	"github.com/Finschia/finschia-sdk/baseapp"
+	simappparams "github.com/Finschia/finschia-sdk/simapp/params"
+	sdk "github.com/Finschia/finschia-sdk/types"
+	"github.com/Finschia/finschia-sdk/types/module"
+	simtypes "github.com/Finschia/finschia-sdk/types/simulation"
+	settlementsimulation "github.com/Finschia/finschia-sdk/x/or/settlement/simulation"
+	"github.com/Finschia/finschia-sdk/x/or/settlement/testutil/sample"
+	"github.com/Finschia/finschia-sdk/x/or/settlement/types"
+	"github.com/Finschia/finschia-sdk/x/simulation"
+)
+
+// avoid unused import issue
```

Comment:
Does that mean these could be used in the future?

Reply:
This comment is generated by ignite.